docs/DeveloperGuide.adoc

```diff
@@ -529,47 +552,48 @@ The new image stored in directory is given a unique name which is formed using h
 
 ==== Reason for this type of implementation
 
-The command is implemented in this manner because of the following reasons
+The command is implemented in this manner because of the following reasons:
 
-* By specifying the index, there is no ambiguity as to who should be assigned the display picture
-* The `PATHOFIMAGE` must be absolute path on the local device to make sure the image is referenced.
-* The image is stored in directory with a unique name to avoid conflict. Hashcode of user's email address is used to maintain uniqueness
-* The task is done using CLI and follows as NFR
+* By specifying the index, there is no ambiguity as to who should be assigned the display picture.
+* The `PATHOFIMAGE` must be an absolute path on the local device to make sure the image is referenced.
+* The image is stored in directory with a unique name to avoid conflict. Hashcode of user's email address is used to maintain uniqueness.
+* The task is done using CLI to follow project requirements.
 * The image is stored in jpg format to ensure maximum compatibility.
 
 ==== Design Considerations
 
 **Aspect:** At what stage should the image be read and stored +
-**Alternative 1 (current choice):** Invoke 'ReadAndStoreImage' from Logic Component  +
-**Pros:** We will be able to get the contact's address for unique hash code  +
-**Cons:** Hard for new developers to understand the sequence diagram and flow. +
-**Alternative 2:** Invoke 'ReadAndStoreImage' from Model Component +
+**Alternative 1 (current choice):** Invoke 'ReadAndStoreImage' from Logic component.  +
+**Pros:** We will be able to get the contact's address for unique hash code.  +
+**Cons:** Difficult for new developers to understand the sequence diagram and flow. +
+**Alternative 2:** Invoke 'ReadAndStoreImage' from Model component. +
 **Pros:** Easier for new developers to understand the sequence diagram and maintains event-driven nature. +
-**Cons:** There might be clash in naming of two different images for large database of contacts
+**Cons:** There might be clashes in naming of two different images for large database of contacts.
 
 ---
 
 **Aspect:** How should the image be taken from user +
-**Alternative 1 (current choice):** User has to enter absolute path of image by checking his local device   +
+**Alternative 1 (current choice):** User has to enter the absolute path of image by checking his local device.   +
 **Pros:** Complete CLI process  +
-**Cons:** Might be problematic for user to copy and paste and might result in error of path giving fail command +
-**Alternative 2:** Pop up a `FileChooser` after command is entered +
-**Pros:** Easier for users to mention the correct image at a quick pace +
-**Cons:** Will no longer be a CLI process completely
+**Cons:** Might be problematic for user to copy and paste and might result in error of path giving fail command. +
```

Comment:
Might be problematic for users to copy and paste,  and this might result in an erroneous path given, leading to an invalid command.